src/EnergyPlus/test_ep_as_library.cc

```diff
@@ -64,10 +64,17 @@ int main(int argc, char *argv[])
     StoreMessageCallback(message_callback_handler);
     StoreProgressCallback(progress_callback_handler);
 
+    int status(EXIT_FAILURE);
     if (argc < 2) {
         std::cout << "Call this with a path to run EnergyPlus as the only argument" << std::endl;
-        return 1;
+        return EXIT_FAILURE;
     } else {
-        EnergyPlusPgm(argv[1]);
+        status = EnergyPlusPgmReturnCodes(argv[1]);
     }
+    if (!std::cin.good()) std::cin.clear();
+    if (!std::cerr.good()) std::cerr.clear();
+    if (!std::cout.good()) std::cout.clear();
+    std::cerr << "THIS LINE SHOULD DISPLAY." << std::endl;
```

Comment:
Do we still need these lines here (this one and the next one)? If we do, I suggest changing them to print out "Standard error is still available for use" and "Standard output is still available for use". The context for the addition of these lines are here will fade over time, so it's better to change them to something more self explanatory.

Reply:
I like the suggested changes to the lines. @DeadParrot gave me a preview of the new ObjexxFCL and that did indeed fix the stdin/stdout issue but it may be nice to have code in this example use file that explicitly demonstrates that there is activity after the RunEnergyPlus call (showing that we're not just exiting anymore). That said, happy to go either way on whether we keep the lines or not.